config/crd/bases/awx.ansible.com_awxbackups.yaml

```diff
@@ -90,6 +90,9 @@ spec:
               postgres_image_version:
                 description: PostgreSQL container image version to use
                 type: string
+              postgres_selector:
```

Comment:
i think postgres_selector is not very descriptive

maybe something like `db_management_pod_node_select`?